### PR TITLE
MDCT-2877: SAR Dashboard (State User + Empty View)

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Empty view of the SAR dashboard for a state user, including an alert banner that shows if a user doesn't have an ongoing Work Plan.

Follow-up items / not covered in this pull request:

- Once statusing is in place for the Work Plan, the alert banner will need to be updated to show if a Work Plan is submitted and approved by an admin
- Table contents themselves are covered in a separate ticket

### Related ticket(s)

MDCT-2877

---
### How to test

- Log in as a state user
- Navigate to the `/sar` page

You should see an alert banner ⏰ 

- Navigate to the `/wp` 
- Create a new Work Plan
- Navigate back to the `/sar` page

The alert should be gone! 🪄 

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by britt-mo — COMMENTED** [review]
Screenshot 2023-10-16 at 1 14 55 PM
seems like the banner should be centered above the table?

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx [hunk 1]
```diff
@@ -237,22 +243,34 @@ export const DashboardPage = ({ reportType }: Props) => {
...
       <Link as={RouterLink} to="/" sx={sx.returnLink}>
         <Image src={arrowLeftIcon} alt="Arrow left" className="returnIcon" />
-        Return Home
+        Return home
       </Link>
       {errorMessage && <ErrorAlert error={errorMessage} />}
+      {/* Only show SAR alert banner if the corresponding Work Plan is not approved */}
+      {reportType === ReportType.SAR &&
+        workPlanStatus !== ReportStatus.APPROVED && (
+          <Alert
```
Comment (cassandradanger) [inline services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx:252]: talked to Kim and we should be left-aligning this alert banner to the page content

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx [hunk 2]
```diff
@@ -237,22 +243,34 @@ export const DashboardPage = ({ reportType }: Props) => {
...
-            verbiage={
-              userIsAdmin
-                ? accordion.WP.adminDashboard
-                : accordion.WP.stateUserDashboard
-            }
-          />
-        )}
+        <InstructionsAccordion
+          verbiage={
+            userIsAdmin
```
Comment (cassandradanger) [inline services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx:267]: I think we need different instructions for admin WP and admin SAR

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx (hunk 1 above)
Comment (cassandradanger) [reply services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx:252]: just seeing britt's comment :) 
I bet centering is preferred and what we've done in the past

Comment (karla-vm) [reply services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx]: I just updated that following the Figma, let me ask Kim!

**Comment by karla-vm** [comment]
@britt-mo and @cassandradanger just spoke to Kim and she said the Alert Banner should be left-aligned 💯 could y'all take a look?

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx (hunk 2 above)
Comment (karla-vm) [reply services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx:267]: heya! so the `reportType` index will pull in the correct instructions from the verbiage file, it should work as expected (once I put in the verbiage for admin `SAR`)

**Review by cassandradanger — APPROVED** [review]
lgtm!

**Pull request merged**